toolkits/linear/evals/eval_issues.py

```diff
@@ -0,0 +1,106 @@
+"""
+Evaluation suite for Linear get_issue tool.
+"""
+
+from arcade_evals import (
+    BinaryCritic,
+    EvalRubric,
+    EvalSuite,
+    ExpectedToolCall,
+    tool_eval,
+)
+from arcade_tdk import ToolCatalog
+
+import arcade_linear
+from arcade_linear.tools.issues import get_issue
+
+# Evaluation rubric
+rubric = EvalRubric(
+    fail_threshold=0.85,
+    warn_threshold=0.95,
+)
+
+# Tool catalog
+catalog = ToolCatalog()
+catalog.add_module(arcade_linear)
+
+
+@tool_eval()
+def get_issue_eval_suite() -> EvalSuite:
+    """Comprehensive evaluation suite for get_issue tool"""
+    suite = EvalSuite(
+        name="Get Issue Evaluation",
+        system_message=(
+            "You are an AI assistant with access to Linear tools. "
+            "Use them to help the user get detailed information about Linear issues."
+        ),
+        catalog=catalog,
+        rubric=rubric,
+    )
+
+    # Eval Prompt: Show me complete details for issue API-789
```

Comment:
nit: no need to have these comments, the prompt is already in the `user_message`